README.md

```diff
@@ -77,6 +77,17 @@ It affects the output of:
 
 Indicated by `--task openrpc`. Assembles a corpus of individual OpenRPC documents into a single OpenRPC document.
 
+#### Tags
+* __event__ - When a method is tagged with this then no SDK method is generated. Events are documented in a different section of the module. The app can use the event not through a method call but by doing `Module.listen('eventName')`
```

Comment:
it's true that "no SDK method is generated" but I think that's a detail that I wouldn't put in the very first sentence, since other tags also achieve that. Maybe start with

> When a method is tagged with this then it will be treated as an asynchronous event that receives a success response, and then zero or more asynchronous responses with the same ID. These responses correlate to the event happening.

Feel free to wordsmith.